testdata/session/valid.yaml

```diff
@@ -12,8 +12,8 @@ user:
   commType: tcp
 
 
-contactsType: yaml             
-contactsURL: ./test-contacts.yaml
+idProviderType: yaml
```

Comment:
> One of the general remarks is that, I noticed the term `idProvider` being used in comments in many places. I think it is better to use it as `ID provider`. I have made suggestion for this in a most places, if I have missed any places, please update it there as well. @anagha-ks

SInce the file name was idprovider.go i kept idprovider in all places where it was actually mentioning the file name. [Changed](https://github.com/hyperledger-labs/perun-node/pull/135/commits/d7c5500a7b2664abeae8f449eddbbd7d163a6b49) to ID provider in all places.

Reply:
@anagha-ks Which filename do you refer to here, is it `idprovider.yaml` or something else ? Because my reference was the places where the id provider component was referenced in a comment or error message.

Reply:
Keeping ID provider instead of idprovider whether it is referring to that file(ideprovider.go or idprovider.yaml) also. For example if we take the comments on `Provider struct`, `It generates a cache of all peer IDs in the ID provider file during initialization.` here i was using `idprovider` since it was referring to file. I commented to that general remarks. Any way i have changed as per your suggestion.